src/System Application/App/Edit in Excel/src/Filters/EditInExcelFiltersImpl.Codeunit.al

```diff
@@ -161,6 +169,26 @@ codeunit 1491 "Edit in Excel Filters Impl."
             exit(false);
         end;
 
+        ExternalTableFieldName := NameJsonToken.AsValue().AsText();
+
+        if not ODataJsonPayload.SelectToken(StrSubstNo(FieldPayloadAlNameTok, ExternalTableFieldName), AlNameJsonToken) then begin
+            Session.LogMessage('0000I5T', StrSubstNo(TypeNotFoundTxt, ExternalTableFieldName), Verbosity::Error, DataClassification::SystemMetadata, TelemetryScope::ExtensionPublisher, 'Category', EditInExcelTelemetryCategoryTxt);
+            exit(false);
+        end;
+
+        FieldAlName := AlNameJsonToken.AsValue().AsText();
+
+        PageControlField.SetRange(PageNo, PageNumber);
+        PageControlField.SetRange(ControlName, FieldAlName);
+
+        if PageControlField.IsEmpty() then
+            if not IsKey(PageNumber, ExternalTableFieldName) then begin
+                Session.LogMessage('0000I5U', FieldNotOnThePageTxt, Verbosity::Normal, DataClassification::SystemMetadata, TelemetryScope::ExtensionPublisher, 'Category', EditInExcelTelemetryCategoryTxt);
+                if not FilterErrors.ContainsKey(ExternalTableFieldName) then
```

Comment:
Is this a typo that we check if it contains `ExternalTableFieldName` but then we add `FieldAlName`?

Reply:
@encimita no it's on purpose, because the keys in this dictionary is eventually used in the UI to tell the user what filters we removed, I thought that there was a bigger chance they would enjoy the AL field name instead of the external field name. I guess that the field caption would be ideal actually.

Reply:
Ok fair, but then shouldn't we check if the `FieldAlName` is in the dictionary and then add `FieldAlName`?

Instead of checking if `ExternalTableFieldName` is in the dictionary and then we add something else (`FieldAlName`)?

Reply:
@encimita Yeah for sure, huge bug, ty!